app/controllers/tasks_controller.rb

```diff
@@ -10,14 +10,13 @@ def show
   def create
 
     @task = Task.new(task_params)
-    # byebug
     @task.user = current_user
 
     # Placeholder algorithm
     # @task.algorithm = Algorithm.first
 
     if @task.save
-      csv_data = FileConverter.algorithm_1(task_params[:file].tempfile.path)
```

Comment:
Hey Sebastian, here it's still using algorithm one, I will be wanting to change this soon to ensure the right algorithm is applied.